src/App.js

```diff
@@ -0,0 +1,35 @@
+import { Routes, Route } from 'react-router-dom';
+import './App.css';
+
+import Navbar from './components/Navbar';
+
+import Login from './pages/Login';
+import Registration from './pages/Registration';
+import Main from './pages/Main';
+import Vehicle from './pages/Vehicle';
+import MyReservations from './pages/MyReservations';
+import Reservation from './pages/Reservation';
+import Delete from './pages/Delete';
+import Add from './pages/Add';
+
+function App() {
```

Comment:
- [ ] When a user is logged in, kindly remove the "Sign up" and "Log in" links in the navbar so it's perfectly clear that the user is logged in.

That way, the UX will improve a great margin. 💪 

_Tip:  Check the local storage to see if a user is logged in._

- [ ] I couldn't find a way to logout.  When a user is logged in, kindly make a "log out" button or link so more than one user can be tested.

_Tip:  Remove the local storage to make the user log out._

Reply:
Thanks for the feedback, changes were implemented.